docs/dev/allocation.qmd

```diff
@@ -17,28 +17,28 @@ flowchart TD
     H --> |Yes| D(End)
 ```
 
-If allocation is used in a model, [Allocation structs](https://deltares.github.io/Ribasim/dev/allocation.html#the-allocation-struct) are created.
+If allocation is used in a model, [Allocation structs](#sec-allocation-struct) are created.
```

Comment:
When I click on this link, it doesn't redirect me the the correct section. Is it the same behavior on your local machine?